domains/pathak.json

```
@@ -0,0 +1,9 @@
+{
+   "owner": {
+      "username": "Mehulpathak21"
```

```suggestion
      "username": "Mehulpathak12"
```
typo error of your github username, fortunately I recheck😅😅